.github/workflows/push-to-mirror.yml

```diff
@@ -2,18 +2,19 @@ name: '[mirror] push to mirror'
 
 on:
   push:
-    branches:
-      - master
-    tags:
-      '**'
-  workflow_run:
-    workflows: ["test configuration"]
-    branches: [master]
-    types: 
-      - completed
+    branches: ["master"]
+    tags: ["*"]
```

Comment:
I think '**' is better since it includes hierarchical tags like v0.5/alpha

Reply:
I think we should avoid using `/` in tag names. 

it might break in mysterious ways (e.g. would it parse properly in repository connector, or do we need to escape it?)

I think we should have a sanity check to ensure that tags with `/` should fail the test automatically

ping @AhmetNSimsek

Reply:
> I think we should avoid using '/' in tag names.
> I think we should have a sanity check to ensure that tags with / should fail the test automatically

I agree that we should but in the future, someone might not. So I agree, we should add a fail fast